docs/html/reference/installation-report.md

```diff
@@ -56,6 +56,9 @@ package with the following properties:
   URL reference. `false` if the requirements was provided as a name and version
   specifier.
 
+- `is_yanked`: `true` if the requirement was yanked from the index, but was still
+  selected by pip conform to [PEP 592](https://peps.python.org/pep-0592/#installers).
```

Comment:
```suggestion
  selected by pip in comformance with [PEP 592](https://peps.python.org/pep-0592/#installers).
```

?

Reply:
im no native, but [looks](https://grammarhow.com/conform-with-or-conform-to/) like 'conform to' and 'conform with' are both fine

Reply:
all four seem to occur, `conform to` seems most common on [ngrams](https://books.google.com/ngrams/graph?content=conform+to%2Cconform+with%2Cin+conformance+to%2Cin+conformance+with&year_start=1800&year_end=2019&corpus=en-2019&smoothing=3)

Reply:
Not native either :)